dropbox/session.py

```diff
@@ -41,7 +41,6 @@ def init_poolmanager(self, connections, maxsize, block=False, **_):
             block=block,
             cert_reqs=ssl.CERT_REQUIRED,
             ca_certs=_TRUSTED_CERT_FILE,
-            ssl_version=ssl.PROTOCOL_TLSv1,
```

Comment:
Are you sure this is what you want? From https://docs.python.org/2/library/ssl.html#ssl.wrap_socket:

> The parameter `ssl_version` specifies which version of the SSL protocol to use. … If not specified, the default is [`PROTOCOL_SSLv23`](https://docs.python.org/2/library/ssl.html#ssl.PROTOCOL_SSLv23); it provides the most compatibility with other versions.